packages/js-plugin-auction-house-module/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@metaplex-foundation/js-plugin-auction-house-module",
```

Comment:
I'm assuming this is from another PR. Please remove it from this one.

Reply:
removed